packages/grafana-runtime/src/config.ts

```diff
@@ -198,7 +198,6 @@ export class GrafanaBootConfig implements GrafanaConfig {
   rootFolderUID: string | undefined;
   localFileSystemAvailable: boolean | undefined;
   cloudMigrationIsTarget: boolean | undefined;
-  cloudMigrationFeedbackURL = '';
```

Comment:
I think we should deprecate this before removing it, WDYT?

Reply:
@hugohaggmark This targeting Grafana 12.0, that should be the proper time to make such breaking changes no?

Reply:
Appreciate the suggestion @hugohaggmark, we don't believe this is applicable here however, as this configuration supports a UI element that we are removing entirely in G12.

Reply:
No worries, we try to deprecate things before we remove them in any major versions. Also milestone is set to `11.6` so just making sure that this is only goes out in G12.